Makefile.in

```diff
@@ -26,6 +26,8 @@ TAG ?= $(shell git rev-parse --short HEAD)
 
 ES_VERSION ?= 7.10.0
 
+BUSYBOX_VERSION ?= 1.30
```

Comment:
The variables in this file is meant to be updated by users when they want to change the behaviors of the showcase but the busybox version seems to be unnecessary to update, in users opinion. WDYT?

Reply:
It is better configurable,  whether the user needs to change it or not. I'm ok with both.